bruno.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Consider adding a description field to the JSON file to provide more context about its purpose. [medium]